### PR TITLE
vdsm-tool: Create libvirtd conf file for el9

### PR DESCRIPTION
**Pull request by hbraha**
In el9, we use the monolithic daemon libvirtd,
so we need to create a libvirtd config manually
since it's not shipped by deafult.

### CODE REVIEW

File: lib/vdsm/tool/configurators/libvirt.py [hunk 1]
```diff
@@ -42,6 +42,7 @@
 from vdsm import constants
 
 
+_LIBVIRT_CONF_PATH = "/etc/sysconfig/libvirtd"
```
Comment (tinez) [inline lib/vdsm/tool/configurators/libvirt.py:45]: We already have that path defined under the `LDCONF` key (line ~360) in a better way - it uses the `constants` module. Let's move that definition here, and reuse the new variable in `LDCONF`

File: lib/vdsm/tool/configurators/libvirt.py [hunk 2]
```diff
@@ -99,6 +100,9 @@ def isconfigured():
     """
     Check if libvirt is already configured for vdsm
     """
+    # Creates libvirt conf file if not exists
```
Comment (tinez) [inline lib/vdsm/tool/configurators/libvirt.py:103]: Strictly speaking, this is not really "libvirt configration" - this one can be found in `/etc/libvirt/libvirt.conf`. Let's call it "system-level libvirt config". This applies to the commit message too.

**Review by tinez — CHANGES_REQUESTED** [review]

**Review by tinez — APPROVED** [review]

**Review by almusil — APPROVED** [review]

**Pull request merged**